cmd/acra-server/acra-server.go

```diff
@@ -123,6 +123,7 @@ func main() {
 	tlsDbSNIOld := flag.String("tls_db_sni", "", "Expected Server Name (SNI) from database (deprecated, use \"tls_database_sni\" instead)")
 	tlsDbCert := flag.String("tls_database_cert", "", "Path to client TLS certificate shown to database during TLS handshake (overrides \"tls_cert\")")
 	tlsDbKey := flag.String("tls_database_key", "", "Path to private key of the TLS certificate used to connect to database (see \"tls_database_cert\")")
+	tlsUseClientIDFromCertificate := flag.Bool("tls_client_id_from_cert", false, "Use certificate info as clientID for encryption/decryption when used. Use certificate from connector's connection if acraconnector_tls_transport_enable used or from client's connection if acraconnector_transport_encryption_disable used")
```

Comment:
it switch connector to use TLS as transport instead secure session. but there several ways to get clientID from such communication: from static parameter --clientID or from certificate. If used `tls_client_id_from_cert`, then will be used from certificate. Otherwise should be provided non-empty `--clientID` (previous version).